nginx.conf

```diff
@@ -52,14 +52,36 @@ http {
       # allow for HTML "meta redirects" on pages of sites that have been migrated
       # to Federalist. For context, see https://github.com/18F/federalist-proxy/issues/19.
       #
-      # More extensions can be added inside the parentheses, separated 
+      # More extensions can be added inside the parentheses, separated
       # by | characters, e.g. `\.(cfm|php)$`.
       location ~* \.(cfm)$ {
         add_header Content-Type "text/html";
         # Notice there is no trailing slash on the proxy_pass url here.
         proxy_pass <%= ENV["FEDERALIST_S3_BUCKET_URL"] %>;
       }
 
+      # Enable CORS support.
+      # Based on https://enable-cors.org/server_nginx.html
+      set $cors_origins 'https://s.codepen.io';
+      set $cors_methods 'GET, OPTIONS';
```

Comment:
Why "OPTIONS" as well as "GET"?

Reply:
Because browsers will typically first do a "CORS preflight request" (using the `OPTIONS` verb) to make sure the headers are properly set before doing an actual CORS request.

More info: https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request

Reply:
ah got it kk.